tpg.py

```diff
@@ -52,7 +52,17 @@
 __url__ = 'http://cdelord.fr/tpg/'
```

Comment:
`tpg.py` is generated by the make file. The actual source file is `tpg.pyg`. If only `tpg.py` is modified, changes may be lost when `make` is run.

The PR should contain the same modifications in both `tpg.py` and `tpg.pyg` (modify `tpg.pyg` and run `make`).